src/Command/Configuration.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\BackwardCompatibility\Command;
```

Comment:
This isn't a `Command` as such, it's a value object that holds configuration, maybe this should go in the top level for now?

```suggestion
namespace Roave\BackwardCompatibility;
```

Reply:
I can change it for sure.

My rationale was mostly focused on grouping together things that change together.
Even though the `Configuration` isn't a command it's very connected to commands and putting it closer makes things more explicit (to me at least :sweat_smile:).

It's the kind of class that I'd 100% mark as `private`, had we had that capability in PHP :smile:

Reply:
I've decided to keep it as `Roave\BackwardCompatibility\Configuration\Configuration` since I still think it's a self-contained concept that isn't that important to be on the root level.

Happy to hear other arguments, though.